com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/pipeline/GameObjectBuilder.java

```diff
@@ -139,8 +141,7 @@ public Task<Void> create(IResource input) throws IOException, CompileExceptionEr
 
         for (long hash : uniqueResources.keySet()) {
             IResource genResource = uniqueResources.get(hash);
-            taskBuilder.addOutput(genResource);
-            Task<?> embedTask = project.createTask(genResource);
```

Comment:
the same as in collection. it should be sub task creation to make sure we have all the generated resources connected with its parents